calico/operations/upgrading/kubernetes-upgrade.mdx

```diff
@@ -129,16 +129,15 @@ the output and then re-running the command without --dry-run to commit to the ch
 
    :::
 
-1. Use the following command to initiate a rolling update, after replacing
-   `<manifest-file-name>` with the file name of your {{version}} manifest.
+1. Use the following command to initiate a rolling update.
 
-   ```
-   kubectl replace -f <manifest-file-name>
+   ```bash
+   kubectl apply --server-side --force-conflicts -f
```

Comment:
In this line and many others the manifest file is no longer present.

Reply:
It should be kubectl apply --server-side --force-conflicts -f <manifest-file-name>